src/base/path.cpp

```diff
@@ -69,6 +70,8 @@ namespace
 #endif
 }
 
+static_assert(Stringable<Path>);
```

Comment:
As I said before I would expect to see comment near each of such static_assert that clarifies such an assertion. E.g. "Path should satisfy Stringable concept in order to be stored in settings as string".